src/Amara/Varcon/TranslationProvider.php

```diff
@@ -43,10 +43,6 @@ public function getTranslations($from, $to, $threshold = 80)
     {
         $fileHandle = fopen($this->filePath, 'r');
 
-        if (!$fileHandle) {
-            throw new \RuntimeException(sprintf('Unable to open file: %s', $filename));
```

Comment:
I think fopen will generate an E_WARNING error if the file does not exist, and return false.

It's probably worth doing the file_exists beforehand like you suggested.